src/cascadia/TerminalControl/TermControlAutomationPeer.h

```diff
@@ -24,6 +24,8 @@ Author(s):
 
 #pragma once
 
+#include <wrl.h>

```

Comment:
Just checking, this is also in `LibraryIncludes.h`, right? We want it to show up in the precompiled headers.